src/github/client/github-queries.ts

```diff
@@ -6,3 +6,123 @@ query {
 		}
 	}
 }`
+
+export const getPullRequests = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
```

Comment:
I've moved everything from src/sync/queries.ts here, so our GraphQL queries will be in a single place